packages/zoe/src/contractSupport/ratio.js

```diff
@@ -158,17 +158,26 @@ const divideHelper = (amount, ratio, divideOp) => {
   );
 };
 
-/** @type {ScaleAmount} */
+/**
+ * Divide the amount by the ratio, truncating the remainder.
```

Comment:
an `@example` would be nice. I wonder how to do them concisely...

```js
const a = withAmountUtils(makeIssuerKit('A'));
const double = makeRatio(2n, a.brand, 1n);
t.deepEqual(floorDivideBy(a.make(5n), double), a.make(2n));
```

Reply:
of course, the example is nearly the same as the test you added.

I'd sure like to combine them a la [doctest](https://docs.python.org/3/library/doctest.html) / [updoc](http://erights.org/elang/tools/updoc.html).